crates/wash-lib/src/app.rs

```diff
@@ -26,6 +28,20 @@ pub enum AppManifest {
     SerializedModel(serde_yaml::Value),
     ModelName(String),
 }
+pub struct FetchFailure {
+    pub image: String,
+    pub error_message: String,
```

Comment:
```suggestion
    pub(crate) image: String,
    pub(crate) error_message: String,
```

Let's hide *most* of the implementation from anyone outside for now)! We'll expose an object that they can at best display.